python/ray/_private/parameter.py

```diff
@@ -316,7 +318,8 @@ def wrap_port(port):
                         f"Ray component {comp} is trying to use "
                         f"a port number {port} that is used by "
                         "other components.\n"
-                        f"Port information: {pre_selected_ports}\n"
+                        f"Port information: "
+                        f"{self._format_ports(pre_selected_ports)}\n"
```

Comment:
is this related to this PR?

Reply:
Yes! it is written in the PR description.